pkg/registry/apis/provisioning/jobs/export/worker_test.go

```diff
@@ -0,0 +1,1323 @@
+package export
```

Comment:
I will improve the tests together with the code. Too many situations and client initialization, cloning, etc. 

Some edge cases are not covered yet as it got out of hand.